packages/driver/cypress/e2e/commands/files.cy.js

```diff
@@ -236,12 +254,12 @@ describe('src/cy/commands/files', () => {
         err.code = 'EISDIR'
         err.filePath = '/path/to/foo'
 
-        Cypress.backend.withArgs('read:file').rejects(err)
+        Cypress.backend.rejects(err)
 
         cy.on('fail', (err) => {
           const { fileLog } = this
 
-          assertLogLength(this.logs, 2)
+          assertLogLength(this.logs, 1)
```

Comment:
@chrisbreiding I am not sure why this generates only a single log this time, but the rest of the error checks out? its `2` on the `release/13.0.0` branch and `develop`

Reply:
@ryanthemanuel this is the error we are seeing in CI and I think the change I have previously was omitting the protocol log since the backend args were stubbed out. going to put this assertion back to 2 and re run and will resolve the convo